page.codeberg.friedrich.DND-Dice-Roller.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "page.codeberg.friedrich.DND-Dice-Roller",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "24.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "base": "com.system76.Cosmic.BaseApp",
+  "base-version": "stable",
+  "command": "dnd_dice_roller",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--talk-name=com.system76.CosmicSettingsDaemon",
+    "--filesystem=xdg-config/cosmic:ro"
```

Comment:
needs an exception

Reply:
I opened a pull request for the exeptions file: https://github.com/flathub-infra/flatpak-builder-lint/pull/696

Reply:
restart the build in an hour or so.